helm-charts/yunikorn/templates/cleanup.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: yunikorn-cleanup
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "3"
```

Comment:
My understanding the weight is giving us a determinable order when installing resources.
But why we have the weight=3 for the cleanup?
This is a post-delete hook, which only triggers when we delete the helm chart, correct?

Reply:
Yes, absolutely right. I will remove this annotation. Thanks for this reminder.